misc/scripts/check_ci_log.py

```diff
@@ -43,23 +46,75 @@
 # It may happen that Godot detects leaking nodes/resources and removes them, so
 # this possibility should also be handled as a potential error, even if
 # LeakSanitizer doesn't report anything
+# Example - https://github.com/godotengine/godot/issues/49438#issue-915348285
 
 if file_contents.find("ObjectDB instances leaked at exit") != -1:
-    print("ERROR: Memory leak was found")
+    print('ERROR: Memory leak was found(search for "ObjectDB instances leaked at exit" in CI log)')
```

Comment:
```suggestion
    print('ERROR: Memory leak was found (search for "ObjectDB instances leaked at exit" in CI log)')
```

Same ofor other errors, a space before a parenthesis is important for readability.